### PR TITLE
DOCS: Feature/fix merge confusion plus improvements to storage params doc

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 1]
```diff
@@ -543,6 +544,46 @@ Move a table to a different schema:
...
+
+```
+ALTER TABLE sales SET ACCESS METHOD ao_row;
+```
+
+Change all future partitions of a table to have an access method of `heap`, leaving the access method of current partitions as is:
+
+ALTER TABLE ONLY sales SET ACCESS METHOD heap;
+
+Add a column and change the table's access method, simultaneously:
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:575]: Seems like we can remove "simultaneously" here.

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 2]
```diff
@@ -543,6 +544,46 @@ Move a table to a different schema:
...
+
+```
+ALTER TABLE distributors SET ACCESS METHOD ao_row, ADD column j int;
+```
+
+Add a column and change table storage parameters simultaneously:
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:581]: Same here.

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md [hunk 3]
```diff
@@ -119,7 +119,7 @@ and storage_directive for a column is:
     [blocksize={8192-2097152} ]
 ```
 
-and storage\_parameter for the table is:
+and storage\_parameter for a table or partition is:
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:122]: storage\_parameter -> `<storage_parameter>`

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md [hunk 4]
```diff
@@ -168,7 +171,7 @@ and partition\_element is:
   | [PARTITION <name>] 
      END ([<datatype>] '<end_value>') [INCLUSIVE | EXCLUSIVE]
      [ EVERY ([<datatype>] [<number | >INTERVAL] '<interval_value>') ]
-[ WITH ( <partition_storage_parameter>=<value> [, ... ] ) ]
+[ WITH ( <storage_parameter>=<value> [, ... ] ) ]
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:174]: spacing around = and [

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md [hunk 5]
```diff
@@ -168,7 +171,7 @@ and partition\_element is:
   | [PARTITION <name>] 
      END ([<datatype>] '<end_value>') [INCLUSIVE | EXCLUSIVE]
      [ EVERY ([<datatype>] [<number | >INTERVAL] '<interval_value>') ]
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:173]: <number | >INTERVAL -> <number> | INTERVAL

(I realize this has nothing to do with your changes, but I can't unsee the error here :)

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md [hunk 6]
```diff
@@ -191,25 +194,11 @@ and subpartition\_element is:
   | [SUBPARTITION <name>] 
      END ([<datatype>] '<end_value>') [INCLUSIVE | EXCLUSIVE]
      [ EVERY ([<datatype>] [<number | >INTERVAL] '<interval_value>') ]
-[ WITH ( <partition_storage_parameter>=<value> [, ... ] ) ]
+[ WITH ( <storage_parameter>=<value> [, ... ] ) ]
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:197]: spacing around = and [

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md [hunk 7]
```diff
@@ -472,6 +431,44 @@ SUBPARTITION BY
 SUBPARTITION TEMPLATE
 :   Instead of declaring each subpartition definition individually for each partition, you can optionally declare a subpartition template to be used to create the subpartitions \(lower level child tables\). This subpartition specification would then apply to all parent partitions.
 
+### <a id="storage_parameters"></a>Storage Parameters
+
+The `WITH` clause can specify storage parameters for tables, and for indexes associated with a `UNIQUE` or `PRIMARY` constraint. Storage parameters for indexes are documented on the [CREATE INDEX](CREATE_INDEX.html.md) reference page. Note that you can also set storage parameters on a particular partition or subpartition by declaring the `WITH` clause in the partition specification. The lowest-level partition's settings have priority.
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:436]: on a particular partition -> for a partitcular partition

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md [hunk 8]
```diff
@@ -472,6 +431,44 @@ SUBPARTITION BY
...
 
+### <a id="storage_parameters"></a>Storage Parameters
+
+The `WITH` clause can specify storage parameters for tables, and for indexes associated with a `UNIQUE` or `PRIMARY` constraint. Storage parameters for indexes are documented on the [CREATE INDEX](CREATE_INDEX.html.md) reference page. Note that you can also set storage parameters on a particular partition or subpartition by declaring the `WITH` clause in the partition specification. The lowest-level partition's settings have priority. 
+
+The defaults for some of the table storage options can be specified with the server configuration parameter `gp_default_storage_options`. For information about setting default storage options, see [Notes](#section5).
+
+Greenplum Database supports the following storage parameters for tables and partitions:
+
+:   **appendoptimized** — Set to `TRUE` to create the table as an append-optimized table. If `FALSE` or not declared, the table will be created as a regular heap-storage table.
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:442]: This should be formatted as a regular deflist.  Each item should look roughly like:

appendoptimized
:    Set to `TRUE` to ...

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md [hunk 9]
```diff
@@ -472,6 +431,44 @@ SUBPARTITION BY
...
+
+:   **checksum** — This option is valid only for append-optimized tables \(`appendoptimized=TRUE`\). The value `TRUE` is the default and enables CRC checksum validation for append-optimized tables. The checksum is calculated during block creation and is stored on disk. Checksum validation is performed during block reads. If the checksum calculated during the read does not match the stored checksum, the transaction is cancelled. If you set the value to `FALSE` to deactivate checksum validation, checking the table data for on-disk corruption will not be performed.
+
+:   **compresstype** — Set to `ZLIB` \(the default\), `ZSTD`, `RLE_TYPE`, or `QUICKLZ`1 to specify the type of compression used. The value `NONE` deactivates compression. Zstd provides for both speed or a good compression ratio, tunable with the `compresslevel` option. QuickLZ and zlib are provided for backwards-compatibility. Zstd outperforms these compression types on usual workloads. The `compresstype` option is only valid if `appendoptimized=TRUE`.
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:450]: 1 -> <sup>1</sup>

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md [hunk 10]
```diff
@@ -472,6 +431,44 @@ SUBPARTITION BY
...
+
+    > **Note** 1QuickLZ compression is available only in the commercial release of VMware Greenplum.
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:452]: sup>1</sup>

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md (hunk 9 above)
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:450]: or a good -> and a good

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md [hunk 11]
```diff
@@ -472,6 +431,44 @@ SUBPARTITION BY
...
+
+    The value `RLE_TYPE`, which is supported only if `orientation`=`column` is specified, enables the run-length encoding \(RLE\) compression algorithm. RLE compresses data better than the Zstd, zlib, or QuickLZ compression algorithms when the same data value occurs in many consecutive rows.
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:454]: These follow-on deflist paragraphs should start with:

:   

(A colon and 4 spaces).

**Review by dyozie — CHANGES_REQUESTED** [review]

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md (hunk 5 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:173]: This occurs in multiple places. Are you sure it's incorrect?

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md (hunk 10 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:452]: I recognize this as related to footnotes in Markdown but I don't know exactly where you intend this to go.

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md (hunk 11 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:454]: I need you to explain this comment. You've just requested above to do a colon followed by a single space. Now you're talking about four spaces. Which is it?

**Review by dyozie — APPROVED** [review]

**Pull request merged**